### PR TITLE
47 new actions

### PR DESCRIPTION
**Pull request by RjImpx**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Hi @RjImpx! Our certification engineer, Srikanth, is currently on emergency leave. Please expect a delay with your PR. He will return January 31st, 2022. We apologize for any inconveniences.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@RjImpx It seems a pretty big update, the updates you have added to the swagger have increased the file size and even Github is not able to show the changes made in the swagger.

I have checked your swagger size and it is 1058 KB. Currently, we are not allowing the 1 MB+ swaggers due to infrastructure limitations. 

But you can do split the connector into two based on the type of actions and triggers and submit the new one for certification.

Please let me know if you have any questions.

**Comment by RjImpx** [comment]
> 

@sriyen-msft we removed some new actions so that we are within the swagger file limitation and we also kept room for future growth. Please review. thank you

File: certified-connectors/Impexium/apiDefinition.swagger.json [hunk 1]
```diff
@@ -3550,111 +3529,1536 @@
...
+        "operationId": "Add-Individual",
+        "parameters": [
+          {
+            "name": "createUser",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "False",
+            "description": "Choose True if a user account should be created when the individual is created.",
+            "enum": [
```
[the remark this answers is not in the document]
Comment (RjImpx) [reply certified-connectors/Impexium/apiDefinition.swagger.json]: @v-eisljami @sriyen-msft Based on your feedback, we changed the enum to bool. please review

**Comment by v-eisljami** [comment]
Hey @RjImpx. I hope you are doing well. We have some Swagger validation issues that we wanted to go thru with you. Im sending the error list with error details for the swagger validation. Can you please review them?

##[error]The list of enum values has been removed from the output field. ValuePath: operation/List-of-Customer-Relationships/responses/200/schema/dataList/items/allowPrimary/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/List-of-Customer-Relationships/responses/200/schema/dataList/items/canPurchaseForOrganization/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/List-of-Customer-Relationships/responses/200/schema/dataList/items/canManagePAC/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/List-of-Customer-Relationships/responses/200/schema/dataList/items/canManageOrganization/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Organization-Inactive-Memberships/responses/200/schema/inheritedMembershipBenefits/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Individual-Inactive-Memberships/responses/200/schema/inheritedMembershipBenefits/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/List-All-Awards/responses/200/schema/dataList/items/allowPublicNomination/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Organizations-Relationships/responses/200/schema/dataList/items/relatedToCustomer/memberships/items/inheritedMembershipBenefits/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Organizations-Relationships/responses/200/schema/dataList/items/relatedToCustomer/showInDirectory/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Organizations-Relationships/responses/200/schema/dataList/items/isPrimary/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Organization/responses/200/schema/primary/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Organization/responses/200/schema/showInDirectory/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Organization/responses/200/schema/isPreferredShipping/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Organization/responses/200/schema/isPreferredBilling/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Organization/responses/200/schema/isBadAddress/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Upcoming-Events/responses/200/schema/dataList/items/publicEvent/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Organization-Active-Memberships/responses/200/schema/items/inheritedMembershipBenefits/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-All-Events/responses/200/schema/dataList/items/publicEvent/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Individual-Active-Memberships/responses/200/schema/items/inheritedMembershipBenefits/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-All-Event-Registrations-Information-for-an-Individual/responses/200/schema/dataList/items/event/publicEvent/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-All-Event-Registrations-Information-for-an-Individual/responses/200/schema/dataList/items/event/freeEvent/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Individuals-Relationships/responses/200/schema/dataList/items/relatedToCustomer/memberships/items/inheritedMembershipBenefits/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Individuals-Relationships/responses/200/schema/dataList/items/relatedToCustomer/showInDirectory/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Get-Individuals-Relationships/responses/200/schema/dataList/items/isPrimary/enum Source: NewSwagger
##[error]The type has been changed for the parameter. Existing flows will continue to send the parameter with the old type. ValuePath: operation/Add-Individual/parameter/createUser/type Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Individual/responses/200/schema/primary/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Individual/responses/200/schema/showInDirectory/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Individual/responses/200/schema/isPreferredShipping/enum Source: NewSwagger
##[error]The list of enum values has been removed from the output field. ValuePath: operation/Add-or-Update-Address-to-Individual/responses/200/schema/isPreferredBilling/enum Source: NewSwagger

**Comment by RjImpx** [comment]
Hi,
We changed the enum option as suggested during code review. We have tested the flows and it works.

[Visit Billhighway]
Raaj Jeeth
VP, Product Development
Phone: 877.486.8220
[https://cdn.billhighway.com/images/bh-gb+signature/facebook.png]<https://www.facebook.com/impexium>[https://cdn.billhighway.com/images/bh-gb+signature/twitter.png]<https://twitter.com/impexium>[https://cdn.billhighway.com/images/bh-gb+signature/linkedin.png]<https://www.linkedin.com/company/impexium>[https://cdn.billhighway.com/images/bh-gb+signature/rss.png]<https://www.impexium.com/>

**Comment by sriyen-msft** [comment]
@RjImpx Sorry for the delay in response, Seems the swagger size is still huge as it is not even able to load in the Github review page. Can you please make sure that the swagger size is below 1MB. Thanks

**Comment by RjImpx** [comment]
> 

Hi @sriyen-msft , we checked the file size and it is at 780 K
Swagger
b

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**